model/src/main/proto/exploration.proto

```diff
@@ -253,6 +253,13 @@ message UserAnswer {
   string content_description = 5;
 }
 
+message HintState {
```

Comment:
I'm thinking that this might better fit in its own proto file (maybe something like 'ui_state.proto') to specifically isolate the domain layer objects from those that can be persisted for saved instance states. What do you think?

Reply:
Sounds good. In case we have future bugs related to configuration changes we can include them here